### PR TITLE
Document backup duration format

### PR DESCRIPTION
**Pull request by LLParse**
https://github.com/rancher/rancher/issues/6295

### CODE REVIEW

File: infra-templates/k8s/12/README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 ## Upgrade Kuberenetes Stack:  WARNING
 When upgrading to this version of Kubernetes, rolling back to the previous version is not supported.
+
+## Duration Format
+
+A duration string is a possibly signed sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
```
Comment (cloudnautique) [inline infra-templates/k8s/12/README.md]: I find this is difficult to understand. What does a negative number mean? I think if we tell the user:
1. Valid formats with examples (expanding on whats already there maybe use the default 15m00s or whatever)
2. How to disable backups or set to a minimum.

Comment (LLParse) [reply infra-templates/k8s/12/README.md]: I just copied the [official description](http://golang.org/pkg/time/#ParseDuration), admittedly it is very generic and not tailored to our usage..

Changes imminent

**Comment by LLParse** [comment]
@cloudnautique Ready for review

**Comment by cloudnautique** [comment]
LGTM

**Comment by vincent99** [comment]
Microsecond backups doesn't sound very useful...

**Comment by cloudnautique** [comment]
We could not mention them, or hope that people do 30000us or greater.... They are technically allowed.

**Pull request merged**